storage/aptosdb/src/pruner/state_store/mod.rs

```diff
@@ -19,11 +19,11 @@ use std::sync::{
 #[cfg(test)]
 mod test;
 
-pub const STATE_STORE_PRUNER_NAME: &str = "state store pruner";
+pub const STATE_MERKLE_PRUNER_NAME: &str = "state merkle pruner";
```

Comment:
nit: generally prefer to not have space in the name for this and the ledger pruner. Using underscore or CamelCase seems to be better.

Reply:
Done